Packs/MicrosoftDefenderAdvancedThreatProtection/Integrations/MicrosoftDefenderAdvancedThreatProtection/MicrosoftDefenderAdvancedThreatProtection.py

```diff
@@ -1187,17 +1187,18 @@ def unisolate_machine(self, machine_id, comment):
         }
         return self.ms_client.http_request(method='POST', url_suffix=cmd_url, json_data=json_data)
 
-    def get_machines(self, filter_req, page_size=None, page_num=None):
+    def get_machines(self, filter_req, page_size='', page_num=''):
         """Retrieves a collection of Machines that have communicated with Microsoft Defender ATP cloud on the last 30 days.
 
         Returns:
             dict. Machine's info
         """
         cmd_url = '/machines'
         params = {'$filter': filter_req} if filter_req else {}
-        if page_size:
+
+        if page_size and page_num:
```

Comment:
What if the user only uses `page_size`?
```suggestion
        if page_size:
```

Reply:
The way it is implemented now both are default values in the yml file. 
If this command will run it will actually always be True. 

There is another call to this function from another command in that case both of the arguments will get the default value 'empty string' and not enter the 'if' statment.